crates/sui-core/src/authority/authority_store_pruner.rs

```diff
@@ -139,9 +139,15 @@ impl AuthorityStorePruner {
                 tokio::select! {
                     _ = prune_interval.tick() => {
                         info!("Starting pruning of objects table");
-                        let perpetual_db = perpetual_db.clone();
-                        let num_pruned = Self::prune_objects(num_versions_to_retain, perpetual_db);
+                        let num_pruned = Self::prune_objects(num_versions_to_retain, perpetual_db.clone());
                         info!("Finished pruning with total object versions pruned = {}", num_pruned);
+                        let min_key = ObjectKey(ObjectID::ZERO, SequenceNumber::MIN);
+                        let max_key = ObjectKey(ObjectID::MAX, SequenceNumber::MAX);
+                        if let Ok(()) = perpetual_db.objects.compact_range(&min_key, &max_key) {
```

Comment:
can we record how long this takes in log and/or metrics?

Reply:
Sure let me send a trivial follow up PR. Thanks

Reply:
Added duration as suggested